proposals/lip-update_genesis_block_schema_and_processing.md

```diff
@@ -0,0 +1,169 @@
+
+#### Genesis state Initialization
+
+At this stage, the genesis state initialization logic for all registered modules is executed. 
+For this purpose, registered modules can specify processing logic considering the information in their corresponding entry of the `assets` property. 
+Typically, modules will perform format and data consistency checks of their respective element in `assets` and then initialize their state according to the data provided in it. 
+However, modules should not call protocol logic of other modules as the state of the respective module may not be initialized.
+
+#### Genesis state finalization
+
+At this stage, the genesis state finalization logic for all registered modules is executed. 
+As part of the genesis state finalization logic modules may call exposed functions from other modules to cross-check the state information integrity and/or complete their own state. 
+When this step is completed, there should be a valid initial state for our blockchain ready to process state transitions implied by the next block.
+
+#### Result verification of the genesis block
+
+Finally, the verifications for block header properties are performed for which access to the state store is required. 
+For example, in the case of `stateRoot`, it requires the genesis state to be final before being checked.
+
+## Specification
+
+In this section, we specify the schema of the genesis block and its validity and execution rules.
+
+### Constant
+
+| Name          | Type    | Value       | Description |
+| ------------- |---------| ------------| ------------|
+| `EMPTY_HASH`  | bytes   | SHA-256("") | Hash of empty bytes.|
+
+### Processing stages of the genesis block, block assets, and block header
+
+As introduced in [the Rationale section](#processing-of-the-genesis-block), the processing of the genesis block is performed in four different stages:
+
+* _Static validation of the genesis block_: The stateless checks to ensure the structure of the genesis block are performed. 
+Also, the properties in the block header that do not require access to the state store are checked. 
+These checks are defined in the sections below.
+* _Genesis state initialization_: The genesis state initialization logic for the registered modules is executed. 
+* _Genesis state finalization_: The genesis state finalization logic for the registered modules is executed.  
+* _Result verification of the genesis block_: The block header properties that require access to the state store are verified as specified below.
+
+### Genesis block
+
+#### JSON schema
+
+The genesis block schema is the same as the one defined in [Update block schema and block processing LIP][BlockLIPschema].
+
+#### Validation
+
+The genesis block is validated in the static validation stage as follows:
+
+* _Static validation of the genesis block_:
+    * Check that the `payload` property is set to its default value, i.e., empty array.
```

Comment:
```suggestion
    * Check that the `transactions` property is set to its default value, i.e., empty array.
```

Reply:
Here we could otherwise remove this check. Reason being: we already check that the `transactionsRoot` is the `EMPTY_HASH` below. Which serves the same purpose. As you prefer.